vazhin/app.js

```diff
@@ -0,0 +1,283 @@
+                "You're amazing!",
+                "You're the best",
+                "Well, that was impressive.",
+                "It's possible! don't forget that.",
+                "amazing job 💯"
+            ]
+            const motivatingWordsNode = document.createElement('p');
+            let randomIndex = Math.floor(Math.random() * motivatingWords.length);
+            motivatingWordsNode.innerText = `${motivatingWords[randomIndex]}`
+            motivatingWordsNode.style.textAlign = "center";
+            motivatingWordsNode.classList.add("motivating");
+
+            if (completedBtnClicked == false) {
+
+                task.innerHTML = `<strike>task: ${arrOfTasks[i].task}</strike>`;
+                priority.remove();
+                deadline.remove();
+                todoDiv.insertBefore(motivatingWordsNode, todoDiv.childNodes[1]);
+                completedBtn.innerHTML = '<i class="fas fa-check"></i>';
+                completedBtnClicked = true;
+                return;
+            }
+            else if (completedBtnClicked == true) {
+
+                task.innerHTML = `task: ${arrOfTasks[i].task}`;
+                completedBtn.innerHTML = '<i></i>';
+                document.querySelector('.motivating').remove();
+                todoDiv.insertBefore(priority, todoDiv.childNodes[1]);
+                todoDiv.insertBefore(deadline, todoDiv.childNodes[2]);
+                completedBtnClicked = false;
+                return;
+            }
+        })
+
+        editBtn.addEventListener("click", function () {
+            inEditMode = true;
+            indexOfInEditTask = i;
+
+            addBtn.innerHTML = "edit task"
+            taskText.value = arrOfTasks[i].task;
+            priorityInput.value = arrOfTasks[i].priority;
+            deadlineInput.value = arrOfTasks[i].deadline;
+        })
+    }
+}
+
+function cancelTodo() {
+    taskText.value = "";
+    priorityInput.value = "";
+    deadlineInput.value = "";
+    inEditMode = false;
+    addBtn.innerHTML = "add task"
+}
+
+function changeDateFormat() {
+    const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+    const current_datetime = new Date();
+    let formatted_date = months[current_datetime.getMonth()] + " " + current_datetime.getDate() + ", " + current_datetime.getFullYear();
+    return formatted_date;
+}
+
+function startDateTime() {
+    let today = new Date();
+    let h = today.getHours();
+    let m = today.getMinutes();
+    let s = today.getSeconds();
+    m = checkTime(m);
+    s = checkTime(s);
+    todayDate.innerHTML = `${changeDateFormat()} &nbsp; ${h + ":" + m + ":" + s}`
+    let t = setTimeout(startDateTime, 500);
+}
+
+function checkTime(i) {
+    if (i < 10) { i = "0" + i };
+    return i;
+}
+
+function datePicker() {
+    document.querySelector('#datetimepicker1').datetimepicker();
+};
+
+$(document).ready(function () {
```

Comment:
this is the only function where you used jQuery, I would love to know why you opted for jQuery instead of just doing it the same was you've done everything else!

Well done Vazhin! You have amazing skills.

Reply:
Oh that one is from bootstrap date-picker. it was written in jquery so that's why I added those libraries 😄